webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -213,31 +213,36 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
     );
   }
 
-  private getUserHomePlaceSummary(homePlaceId?: string) {
+  private getUserHomePlaceSummary() {
     return this.userSettingsService
       .get()
-      .then((userSettings:any) => {
-        const facilityId: string = homePlaceId ?? userSettings.facility_id;
+      .then((userSettings: any) => {
+        const facilityId = Array.isArray(userSettings.facility_id)
```

Comment:
HI @latin-panda give me your quick thought on this. This is what I meant by treating all `facility_id` as arrays. If it is a string I typecast it here to treat all as arrays in the component. 

Any concerns about that ?

Reply:
It's okay.

Does the ternary fit in 1 line? It's better seeing like that

Reply:
Maybe you can make the code flat with async?

Reply:
It doesn't fit in 1 line. I will try the async